rotkehlchen/exchanges/bybit.py

```diff
@@ -132,6 +136,19 @@ def __init__(
         }
         self.is_unified_account = False
         self.history_events_db = DBHistoryEvents(self.db)
+        self.five_letter_assets = set()
+        self.six_letter_assets = set()
+
+        with GlobalDBHandler().conn.read_ctx() as cursor:
+            cursor.execute(
+                'SELECT value FROM location_asset_mappings WHERE location IS ? OR location IS NULL;',  # noqa: E501
```

Comment:
I believe you can do this

```suggestion
                'SELECT value FROM location_asset_mappings WHERE (location IS ? OR location IS NULL) AND LENGTH(value) >= 5;',  # noqa: E501
```